setup_rucio.py

```diff
@@ -224,4 +224,10 @@ def get_file_list(self):
     install_requires=requires,
     extras_require=extras_require,
     dependency_links=depend_links,
+    entry_points = {
+    'console_scripts':
+        ['rucio-reaper=rucio.daemons.reaper.clis.reaper:main',
```

Comment:
The actual locations of this clis we should still think about. Like this it feels very fragmented.
Do I understand right that everything in bin/* would go away as it is automatically created when the package is installed? This has some implications in a test environment where you are just working from source and do not install the package. Some of the tests using the CLI directly also have to be changed possibly.

Reply:
I got some hesitation to put them in `rucio/daemons/clis` but it looks fine like that imi. Yes for the tests. The test `test_bin_rucio.py` should be rewritten to not spawn processes or we change our tests to install rucio with python setup.py at the beginning which will validate setup.py and the packaging as well.

Reply:
I think it could make sense to have all these centrally in `lib/rucio/clis`
I dont't like `lib/rucio/client/clis` a lot as we established `client/` as the location for the python apis.

Reply:
It's a **convention** to decide imo. Several options:
1. All entry point source files(daemon+clients) go in `lib/rucio/clis`. It is quiet similar to what we have today as all scripts are centralized in `rucio/bin`.

2. The client ones in `lib/rucio/client/clis` and the daemon ones in `lib/rucio/daemons/clis`.

3. The client ones can also go in `lib/rucio/client/clis`  and the daemons one in each daemon directory `lib/rucio/daemons/{daemon name}/clis`. But the cons is the fragmentation you mentioned and I agree with it. On the other hand, it looks cleaner and self-contained.

4. You name it.

I prefer option 2.

Reply:
I prefer 1 a little bit over 2 for the reason that all the clis are in one directory. To me it feels weird that we point people to the API directory `lib/rucio/client`, when they ask for the rucio APIs and then there is another directory in there which has nothing to do with the API. 
For 1 we could still do a `daemons/` and `clients/` - but I think all in one is fine as well.
3 and 4 I don't like 😄

Reply:
yup, another option might be 4) `lib/rucio/clis/daemons/` and `lib/rucio/clis/clients/` which I prefer now.

Reply:
Agreed, let's do it like this.